docs(concepts): improve concept pages for elements products with links to relevant docs MTA-1453 (#74)
scaleway/docs-content: compute/bare-metal-cloud/concepts.mdx

## Flexible IP

Flexible IP addresses are public IP addresses that you can hold independently of any Bare Metal server. Flexible IP addresses can be [attached to and detached from](/compute/bare-metal-cloud/how-to/attach-detach-flexible-ip/) any of your Bare Metal servers as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users.

</Concept>

<Concept>

## Image

> An image is a file that contains the OS, application data, and any files that might be related to your applications. If you don’t know yet which image you want to run on the Bare Metal server, you can order the server without preinstalled image.

Reviewer: ```suggestion
An image is a file that contains the OS, application data, and any files that might be related to your applications. If you do not know yet which image you want to run on the Bare Metal server, you can order the server without preinstalled image.
```
